src/modules/bulletproofs/rangeproof_impl.h

```diff
@@ -846,11 +846,7 @@ static int secp256k1_bulletproof_rangeproof_rewind_impl(uint64_t *value, secp256
     secp256k1_scalar_mul(blind, &taux, &z);
     secp256k1_scalar_negate(blind, blind);
 
-    secp256k1_pedersen_ecmult(&rewind_commitj, blind, *value, &value_genp, blind_gen);
-    secp256k1_gej_neg(&rewind_commitj, &rewind_commitj);
-    secp256k1_gej_add_ge_var(&rewind_commitj, &rewind_commitj, &commitp, NULL);
-
-    return secp256k1_gej_is_infinity(&rewind_commitj);
+    return 1;
```

Comment:
What exactly is a return value of 1 supposed to represent after this change?

Reply:
It will return when these conditions are met:
- Proof is properly formatted (`mu`, `taux` aren't zero etc.)
- First 4 bytes of `mu'-mu` are 0 (used to be first 8 bytes)
There is a decent probability that the output belongs to the wallet when it passes this second check, but it can produce some false positives. This is why we have a waterproof check on the grin level (see other PR).

The `gamma*G + v*H == commit` check is removed now, since for the new rewind scheme `gamma` won't be the actual private key.